core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/OutputExpression.scala

```diff
@@ -10,22 +10,27 @@ sealed trait OutputExpressionTarget {
 case object ThisEntityTarget extends OutputExpressionTarget { override val prefix = "this." }
 case object WorkspaceTarget extends OutputExpressionTarget { override val prefix = "workspace." }
 
-case class OutputExpression(target: OutputExpressionTarget, attributeName: AttributeName)
+sealed trait OutputExpression
+case object UnboundOutputExpression extends OutputExpression
+case class BoundOutputExpression(target: OutputExpressionTarget, attributeName: AttributeName, attribute: Attribute) extends OutputExpression
```

Comment:
Updated prefix -> root and changed the parsing location.

`(AttributeName, Attribute)` is consistent with how it's used all over Rawls so I'm keeping those names